src/Identity/IdentityServer/RequestValidators/WebAuthnGrantValidator.cs

```diff
@@ -98,7 +101,8 @@ public async Task ValidateAsync(ExtensionGrantValidationContext context)
         }
 
         var (user, credential) = await _assertWebAuthnLoginCredentialCommand.AssertWebAuthnLoginCredential(token.Options, deviceResponse);
-        UserDecryptionOptionsBuilder.WithWebAuthnLoginCredential(credential);
+        var allCredentials = await _webAuthnCredentialRepository.GetManyByUserIdAsync(user.Id);
+        UserDecryptionOptionsBuilder.WithWebAuthnLoginCredentials(allCredentials);
```

Comment:
@trmartin4 Was there a decision by Auth to not send all of these as part of the sign in and just rely on sync?

If so - I will remove the array changes for the Auth response models. In theory, we could also remove the transports from Auth's flow. 

(cc: @jlf0dev)

Reply:
The only reason to keep sending all down is if we expect the user to sign in with something else and the decrypt using a passkey?

Or do we perform a sync in-between the sign in and the next unlock step?

Reply:
For the record: We decided to handle the array mapping on the client side, causing us to not having to do a breaking change server side.